src/ui/shared/requests/useWalletSecretQuery.ts

```diff
@@ -10,18 +11,22 @@ async function getSecretValue({
   address?: string | null;
   groupId?: string | null;
   seedType: SeedType;
-}) {
+}): Promise<
+  | { seedType: SeedType.privateKey; value: string }
```

Comment:
Could you elaborate a bit on why the `privateKey` is not `LocallyEncoded`?

Reply:
It does, updated :)